napari/layers/points/_tests/test_points.py

```diff
@@ -1593,3 +1594,29 @@ def test_prepare_properties():
     properties, choices = layer._prepare_properties({"aa": [1, 2, 1]})
     assert np.array_equal(properties["aa"], [1, 2, 1])
     assert np.array_equal(choices["aa"], [1, 2])
+
+
+def test_set_face_color_mode_after_set_properties():
+    # See GitHub issue for more details:
+    # https://github.com/napari/napari/issues/2755
+    rng = np.random.default_rng(0)
+    num_points = 3
+    points = Points(rng.random((num_points, 2)))
+
+    points.properties = {
+        'cat': rng.integers(low=0, high=num_points, size=num_points),
+        'cont': rng.random(num_points),
+    }
+
+    with pytest.warns(UserWarning):
```

Comment:
What's the warning here? Could you maybe add a comment about it?

Reply:
Done.

The presence of this warning also makes me think this test doesn't really represent an intended use. @kevinyamauchi : if you have any opinions here or any suggestions to make this test closer to a use of setting `face_color_mode`, that would be great!